asset-transfer-basic/chaincode-java/Dockerfile

```diff
@@ -24,8 +24,8 @@ RUN addgroup --system javauser && useradd -g javauser javauser
 COPY --chown=javauser:javauser --from=GRADLE_BUILD /home/gradle/build/libs/chaincode.jar /chaincode.jar
 COPY --chown=javauser:javauser docker/docker-entrypoint.sh /docker-entrypoint.sh 
 
-ENV PORT $CC_SERVER_PORT
-EXPOSE $CC_SERVER_PORT
+ENV PORT 9999
```

Comment:
?

I haven't tested this but this looks "sus."  

Why set the ARG default above, and then stomp on it here with a literal value?

Reply:
cold light of day, with fresh coffee.. yes that is 'sus'  
I'll circle back round on why that was an issue.